framework/src/test/java/viewpointtest/ViewpointTestAnnotatedTypeFactory.java

```diff
@@ -3,19 +3,33 @@
 import org.checkerframework.common.basetype.BaseAnnotatedTypeFactory;
 import org.checkerframework.common.basetype.BaseTypeChecker;
 import org.checkerframework.framework.type.AbstractViewpointAdapter;
+import org.checkerframework.framework.type.QualifierHierarchy;
+import org.checkerframework.javacutil.AnnotationBuilder;
 
 import java.lang.annotation.Annotation;
 import java.util.Set;
 
+import javax.lang.model.element.AnnotationMirror;
+
 import viewpointtest.quals.A;
 import viewpointtest.quals.B;
 import viewpointtest.quals.Bottom;
+import viewpointtest.quals.Lost;
 import viewpointtest.quals.PolyVP;
 import viewpointtest.quals.ReceiverDependentQual;
 import viewpointtest.quals.Top;
 
+/** The annotated type factory for the viewpointtest type-system. */
```

Comment:
```suggestion
/** The annotated type factory for the Viewpoint Test Checker. */
```

It would be nice to use consistent wording for the different javadocs.